spec/requests/sidenav_pages_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe "Side Navbar" do
+  subject { page }
+
+  describe "Self Assessment Link" do
+    let!(:user) { FactoryGirl.create(:user) }
+    let!(:ac) { FactoryGirl.create(:associate_consultant,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.